checker/pass_context_first_checker.go

```diff
@@ -18,6 +18,26 @@ func NewPassContextFirstChecker(configData interface{}) NodeChecker {
 	return &PassContextFirstChecker{}
 }
 
+// Title implements the NodeChecker interface.
+func (c *PassContextFirstChecker) Title() string {
+	return "Context Argument First"
+}
+
+// Description implements the NodeChecker interface.
+func (c *PassContextFirstChecker) Description() string {
+	return `A function must receive context.Context as its first argument.`
```

Comment:
How about `When a function receives context.Context it must be its first argument.`?

Reply:
Done.